### PR TITLE
Undocumented IOPS return

### PR DESCRIPTION
**Pull request by doticatto**
IOPS is being returned by the module, but simply was not documented. Simply a documentation change.

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

**Review by alinabuzachis — COMMENTED** [review]
@doticatto  Thank you for taking time to work on this. Could you please remove the `.rst` file?

File: docs/community.aws.rds_instance_info_module.rst [hunk 1]
```diff
@@ -957,6 +957,24 @@ Common return values are documented `here <https://docs.ansible.com/ansible/late
                         <div style="font-size: smaller"><b>Sample:</b></div>
                         <div style="font-size: smaller; color: blue; word-wrap: break-word; word-break: break-all;">2017-10-10T04:00:07.434000+00:00</div>
                 </td>
+            </tr>
```
Comment (markuman) [inline docs/community.aws.rds_instance_info_module.rst:960]: I think the docs/ files are autogenerated...

**Comment by doticatto** [comment]
@alinabuzachis I have removed the .rst file changes. Hopefully the checks pass

**Comment by alinabuzachis** [comment]
recheck

**Review by alinabuzachis — APPROVED** [review]

**Comment by tremble** [comment]
Thanks for this update.  A quirk of the way we return these values is that some of them will 'magically' pop up when botocore is updated by Amazon, without any code changing in the module.  This in turn results in things missing in the documentation.

**Review by markuman — APPROVED** [review]
LGTM

**Comment by alinabuzachis** [comment]
recheck

**Pull request merged**